specviz/conftest.py

```diff
@@ -64,5 +64,5 @@ def specviz_gui():
     for testing of specviz gui
     :return:
```

Comment:
@drdavella since we're talking about testing coding style, docstrings should be numpy-styled.